azavea/raster-foundry
# Add org->feature mapping table

## Overview

This PR adds a table for organization->feature mappings.

### Checklist

~~- [ ] Styleguide updated, if necessary~~
~~- [ ] Swagger specification updated, if necessary~~

## Notes
The migration itself isn't terribly interesting, but rather the point of review should probably be whether this DB revision makes sense for the purposes of persisting feature flags per organization.

## Testing Instructions
* ensure that the migrations apply correctly
* ensure that the DB change is reasonable and complete with respect to the goal outlined in #1408

Closes #1408
